composer.json

```diff
@@ -20,14 +20,14 @@
 	],
 	"require": {
 		"php": "~7.1",
-		"symfony/console": "~3.0",
-		"symfony/yaml": "~3.0"
+		"symfony/console": "~3.2|~4.0",
+		"symfony/yaml": "~3.0|~4.0"
 	},
 	"require-dev": {
-		"consistence/coding-standard": "2.0",
-		"jakub-onderka/php-parallel-lint": "0.9.2",
-		"phpstan/phpstan-shim": "0.8.4",
-		"phpunit/phpunit": "6.3.0"
+		"consistence/coding-standard": "^2.0",
+		"jakub-onderka/php-parallel-lint": "^1.0",
+		"phpstan/phpstan-shim": "0.9.2",
+		"phpunit/phpunit": "^7.0"
```

Comment:
It should be also changed in https://github.com/mhujer/yaml-sort-checker/blob/0fcb205d8262f95552b650d105c00c5bf1dd78ba/phpunit.xml.dist#L5

Maybe it's worth switching to how the Doctrine does it - https://github.com/doctrine/doctrine2/blob/357005735833f709750cee59cf37dd5baf77c0a1/phpunit.xml.dist#L16 (`xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd`)

Reply:
Nice trick, I did know about this :-)